docs/source/conf.py

```diff
@@ -0,0 +1,186 @@
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ["_templates"]
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+source_suffix = {
+    ".rst": "restructuredtext",
+    ".txt": "restructuredtext",
+    ".md": "markdown",
+}
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = [
+    "*estimation.*_.rst",  # estimation old files which kept locally
+    "*tests.*",  # remove tests from documentation
+]
+
+# https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=master_doc#confval-master_doc
+master_doc = "index"  # what file includes the root toctree. Needed explicitly for readthedocs.org
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+on_rtd = os.environ.get("READTHEDOCS") == "True"
+if on_rtd:
+    html_theme = "sphinx_rtd_theme"
+else:
+    html_theme = "classic"
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ["_static"]
+
+# # Add content to the navigation side bar:
+html_sidebars = {
+    "**": ["localtoc.html", "sourcelink.html", "searchbox.html", "globaltoc.html"]
+}
+
+
+# -- Custom scripts -------------------------------------------------
+
+
+# Add module's README for each module HTML page
+def add_modules_readme() -> str:
+    def get_rst_file_name_from_package_source(package_source_path: str) -> str:
+        rst_file_name = package_source_path.split(os.sep)
+        rst_file_name = [
+            directory for directory in rst_file_name if not directory.startswith(".")
+        ]
+        rst_file_name += ["rst"]
+        rst_file_name = ".".join(rst_file_name)
+        rst_file_name = os.path.join(source_html_dir, rst_file_name)
+        return rst_file_name
+
+    def get_edited_rast_file(
+        rst_source_path: str, include_text: str, remove_original_header: bool = True
+    ) -> str:
+        with open(rst_source_path, "r") as fh:
+            rst_source_lines = fh.read().splitlines()
+
+        if not any([include_text in line for line in rst_source_lines]):
+            # Add link to readme only if not already exists
+            rst_source_lines.insert(3, include_text)
+            if remove_original_header:
+                rst_source_lines = rst_source_lines[2:]  # remove existing header
+
+        return rst_source_lines
+
+    # source_code_dir = os.path.join("..", "..", "fuse-med-ml")  # causallib source code directory
```

Comment:
I'm not convinced that one would work:

in `causallib` they have `causallib` directory in the repo main dir as you can see [here](https://github.com/BiomedSciAI/causallib/tree/master).
In our case we have `fuse`, `fuseimg` and `fuse_examples`.

It might be that `../../` takes us "above" the repository root directory and in that case indeed `os.path.join("..", "..", "fuse-med-ml")` would be good.

How can we make sure of that? What is the relatively path that this script is being called?

Reply:
Hi @SagiPolaczek , Thanks for catching that! I updated it to use all paths relative to os.getcwd() now. I checked the docs and all readme's are updated correctly. 

Please review the latest code. Thanks!